d2cli/export.go

```diff
@@ -0,0 +1,38 @@
+package d2cli
+
+import (
+	"path/filepath"
+)
+
+type exportExtension string
+
+const GIF = ".gif"
+const PNG = ".png"
+const PPTX = ".pptx"
+const PDF = ".pdf"
+const SVG = ".svg"
+
+var KNOWN_EXTENSIONS = []string{SVG, PNG, PDF, PPTX, GIF}
```

Comment:
this reminds me, we should have an error message for if users try to export to `.ppt`: "D2 does not support ppt exports, did you mean "pptx?"

can you make that an issue

Reply:
Already solved here, no need for a new issue